docs-2.0/3.ngql-guide/3.data-types/4.date-and-time.md

```diff
@@ -1,26 +1,28 @@
 # Date and time types
 
-This topic will describe the `DATE`, `TIME`, `DATETIME`, and `TIMESTAMP` types.
+This topic will describe the `DATE`, `TIME`, `DATETIME`, `TIMESTAMP`, and `DURATION` types.
 
-While inserting time-type property values, except for `TIMESTAMP`, Nebula Graph transforms them to a UTC time according to the time zone specified with the `timezone_name` parameter in the [configuration files](../../5.configurations-and-logs/1.configurations/1.configurations.md). The time-type values returned by nGQL queries are all UTC time.
+## Precautions
 
-!!! note
+- While inserting time-type property values with `DATE`, `TIME`, and `DATETIME`, Nebula Graph transforms them to a UTC time according to the time zone specified with the `timezone_name` parameter in the [configuration files](../../5.configurations-and-logs/1.configurations/1.configurations.md). 
 
-    To change the time zone, modify the `timezone_name` value in the configuration files of all Nebula Graph services.
+  !!! note
+
+        To change the time zone, modify the `timezone_name` value in the configuration files of all Nebula Graph services.
+
+- `date()`, `time()`, and `datetime()` can transform a time-type property with a specified time zone. For example, `datetime("2017-03-04 22:30:40.003000+08:00")` or `datetime("2017-03-04T22:30:40.003000[Asia/Shanghai]")`.
```

Comment:
```suggestion
- `date()`, `time()`, and `datetime()` can convert a time-type property with a specified time zone. For example, `datetime("2017-03-04 22:30:40.003000+08:00")` or `datetime("2017-03-04T22:30:40.003000[Asia/Shanghai]")`.
```

Reply:
it's better to use convert instead of transform, transform would mean the size, shape, color, or some other attribute was changed.
just one question: what will the xxx convert to or from?